### PR TITLE
Fix overlooked address -> peerAddress rename

### PR DESCRIPTION
**Pull request by chrispable**

### CODE REVIEW

**Comment by chrispable** [comment]
Trying to compile some replayed dependency updates in a docker image and came across this, easy fix :).

File: unix.c [hunk 1]
```diff
@@ -612,7 +612,7 @@ enet_socket_receive (ENetSocket socket,
 #ifdef NO_MSGAPI
     // This will ONLY work with a single buffer!
     
-    address -> addressLength = sizeof (address -> address);
+    peerAddress -> peerAddressLength = sizeof (address -> address);
```
Comment (cgutman) [inline unix.c]: Need to update `sizeof (address -> address)` too, right?

**Review by cgutman — CHANGES_REQUESTED** [review]

File: unix.c (hunk 1 above)
Comment (chrispable) [reply unix.c]: Yeah sorry. Was juggling multiple submodules and my commit got silly ill update it

**Comment by chrispable** [comment]
Updated. thanks for the catch

**Comment by cgutman** [comment]
This looks good now.

I noticed though that the author details on your commits look wrong `root <root@localhost>`. Do you want to fix that before I merge?

**Comment by chrispable** [comment]
> This looks good now.
> 
> I noticed though that the author details on your commits look wrong `root <root@localhost>`. Do you want to fix that before I merge?

No. It's because of the silly docker container. I don't care about attribution, it's such a minor fix. Thank you though!

**Comment by cgutman** [comment]
Looks like GitHub fixed up the attribution in the squash anyway.

**Pull request merged**